dashboard/app/controllers/datablock_storage_controller.rb

```diff
@@ -225,6 +232,23 @@ def clear_all_data
     render json: true
   end
 
+  ##########################################################
+  #   Project Use Datablock Storage API                    #
+  ##########################################################
+
+  def use_datablock_storage
+    ProjectUseDatablockStorage.set_data_block_storage_for!(params[:channel_id], true)
+    render json: true
+  end
+
+  def use_firebase_storage
+    ProjectUseDatablockStorage.set_data_block_storage_for!(params[:channel_id], false)
+    render json: true
+  end
```

Comment:
We should add our TODO (can't remember exact) for post-merge cleanup/removal to these